config/branch/master.ini

```diff
@@ -6,3 +6,14 @@ deploy_url = ${dev_deploy_repo_url}/${repo}
 
 # Master images should always follow the master ostree.
 stable_branch = ${build:branch}
+
+[flatpak-remote-flathub]
+# Don't install the stable version of Kolibri
+apps_del =
+  org.learningequality.Kolibri
+
+[flatpak-remote-flathub-beta]
+# Instead, install the beta version of Kolibri
+enable = true
+apps_add =
+  org.learningequality.Kolibri//beta
```

Comment:
Is the `//beta` suffix needed?

I see that we define a `default_branch` for regular Flathub and eos-apps. Should we be setting that in the `defaults.ini` definition of flathub-beta?